src/main/scala/fi/oph/koski/luovutuspalvelu/LuovutuspalveluServlet.scala

```diff
@@ -52,6 +60,26 @@ class LuovutuspalveluServlet(implicit val application: KoskiApplication) extends
     }()
   }
 
+  post("/hetut") {
+    implicit val MaxHetus: Int = 1000
```

Comment:
Implicit perus-tyypit (Int, String) on yleensä huono idea, koska se voi vahingossa päätyä jonkin toisen Int:in tilalle joka tarkoittaa ihan muuta. Tässä tapauksessa helpointa varmaan siirtää MaxHetus tuonne parseHetuRequestV1 funkkariin?